src/librustc/middle/intrinsicck.rs

```diff
@@ -111,7 +111,7 @@ impl<'a, 'gcx, 'tcx> ExprVisitor<'a, 'gcx, 'tcx> {
                 }
                 Err(LayoutError::Unknown(bad)) => {
                     if bad == ty {
-                        format!("size can vary")
```

Comment:
I'm not sure if there's much point to this branch myself; I can't come up with an example that would follow it.